tools/copydlldeps.md

```diff
@@ -45,6 +45,24 @@ if [ ! $( echo $compiler | grep -q "shared" ) ]; then
 fi
 ```
 
+Test & example
+==============
+
+Let us compile password manager [KeePassX](https://www.keepassx.org) in
+shared mode (i686-w64-mingw32.shared) and gather all its DLL dependencies.
+
+```bash
+$ make keepassx MXE_PLUGIN_DIRS=plugins/apps \
+    MXE_TARGETS='i686-w64-mingw32.shared'
+$ ./tools/copydlldeps.sh --copy --destdir keepassx-bundle \
+    --infile usr/i686-w64-mingw32.shared/bin/KeePassX.exe \
+    --recursivesrcdir usr/i686-w64-mingw32.shared/
+$ cp -r usr/i686-w64-mingw32.shared/bin/{KeePassX.exe,share} keepassx-bundle
```

Comment:
@starius - could not get keepassx started from git - Unknown CMake command "get_git_head_revision". Get me started and I will test.
Review: every qt application seems to need 
```bash
--enforce usr/i686-w64-mingw32.shared/qt5/plugins/platforms/
```  
as an argument for the application to start. In my new version provided, it would then be
```bash
--enforcedir usr/i686-w64-mingw32.shared/qt5/plugins/platforms/
```